shared/middleware/helpers/bucket_helper.rb

```diff
@@ -24,7 +24,7 @@ def category_from_file_type(extension)
     mime_type = Sinatra::Base.mime_type(extension)
     if mime_type == 'application/pdf'
       'pdf'
-    elsif ['.doc', '.docx'].include? extension
```

Comment:
The `else` block will already return `'text'` for a `.txt` file (as it will return `'image'` for .jpg, etc). Are you sure you want it to be `doc` instead?